pulp_container/app/viewsets.py

```diff
@@ -648,7 +650,7 @@ class ContainerPushRepositoryViewSet(TagOperationsMixin, ReadOnlyRepositoryViewS
                 "action": ["retrieve"],
                 "principal": "authenticated",
                 "effect": "allow",
-                "condition": "has_model_or_obj_perms:container.view_containerpushrepository",
+                "condition": "has_namespace_or_obj_perms:container.view_containerpushrepository",
             },
             {
                 "action": ["tag", "untag"],
```

Comment:
should this be `has_namespace_or_obj_perms:container.modify_content_containerpushrepository`